python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -221,9 +303,8 @@ def __init__(self, feat_data: List[OnDiskFeatureData]):
                     torch.load(spec.path), metadata=metadata
                 )
             elif spec.format == "numpy":
-                mmap_mode = "r+" if not spec.in_memory else None
-                features[key] = TorchBasedFeature(
-                    torch.as_tensor(np.load(spec.path, mmap_mode=mmap_mode)),
+                features[key] = DiskBasedFeature(
```

Comment:
Do we already have unit test to cover this case?

Reply:
Added it.